server/server.js

```diff
@@ -10,6 +10,7 @@ const passport = require('./strategies/user.strategy');
 
 // Route includes
 const userRouter = require('./routes/user.router');
+const templateRouter = require('./routes/template.router');
```

Comment:
lets change template router to cardRouter to be more specific about what the router is doing.
```suggestion
const cardRouter = require('./routes/card.router');
```